### ISSUE TITLE
AR6-WG3 database connection

### ISSUE THREAD

**Issue by chrisroadmap** [GranthamImperial/silicone#148]
I'd find it useful to add a utility function to download the AR6 database, alongside that currently available for SR1.5:

```
def download_or_load_sr15(filename, valid_model_ids="*"):
...
```

which I guess should be possible using the `ar6-public` connection?

https://pyam-iamc.readthedocs.io/en/stable/tutorials/iiasa_dbs.html

**Comment by znicholls** [comment]
I'd put this as out of scope for silicone: it's an infilling package, not a downloading one (but the scope has already crept and I'm not the lead dev). I think there is a good question to be asked about where to put this sort of utility stuff. My suggestions: a gist or we create a specific package e.g. `scenario_database_utils` where we can house these things and test them without bloating other packages

**Comment by Rlamboll** [comment]
I agree that this sounds useful but is not optimally placed here. I have created an issue in pyam suggesting that we add this tool there.

**Comment by chrisroadmap** [comment]
Nice, thanks for considering 👍
